_rfcs/rfc-0002.md

```diff
@@ -28,11 +28,11 @@ This process will be reviewed with a plan to become an official process in Augus
 
 # Motivation
 
-Put simply, as a society we want to make as big a difference as possible in the lives of Edmonton developers and the broader community. Working Groups allow all Dev Edmonton Society members to quickly and effectively perform interesting work of their choosing.
+As a society we want to improve the lives of Edmonton developers as much as possible, enriching the broader community through them. Working Groups allow all Dev Edmonton Society members to quickly and effectively perform interesting work of their choosing.
 
-As well Working Groups keep the work of DES open, collaborative, and welcoming to members of all abilities, interest, and availability.
+Working Groups keep the work of DES open, collaborative, and welcoming to members of all abilities, interest, and availability.
 
-DES provides these Working Groups with resources, a platform, and shared experience to make this possible then steps back to allow members to do the work.
+DES provides Working Groups with resources, a platform, and shared experience to empower members.
```

Comment:
I thought the removed language here tended to distract me while reading rather than helping me understand what was going on. I may have missed the mark with some language.

Reply:
Always like seeing things get shorter @jfeaver. Thanks!

I might tweak the second sentence a little as I want to emphasize that it's the Working Group members we want to let do most of the work, but I agree that the way it was written before was distracting.